modules/configuration/js/configuration_helper.js

```diff
@@ -71,21 +71,24 @@ $(function () {
         e.preventDefault();
 
         var form = $(this).serialize();
```

Comment:
var form = $(this).serialize();
replace to:
`const form = $(this).serialize();`

Reply:
I'm not really looking to modernize the JS code in this PR, especially since it isn't Reactified right now. I think this change and your below suggestion about `console.log` can be addressed in another `Cleanup` PR. I'd like to keep this work focused on the security feature that this adds.

Reply:
@johnsaigle okay, I just saw one of the changes replacing var to let and thought to comment about this one as well. You can mark my suggestions as resolved.